broker/config.go

```diff
@@ -0,0 +1,26 @@
+package broker
+
+import (
+	"github.com/Percona-Lab/percona-dbaas-cli/dbaas"
+)
+
+const (
+	pxcServiceID     = "pxc-service-broker-id"
+	pxcServiceName   = "percona-xtradb-cluster"
+	psmdbServiseID   = "percona-server-for-mongodb"
```

Comment:
```suggestion
	psmdbServiceID   = "percona-server-for-mongodb"
```
Hi folks. Here is the "typo" 😊. And in other places where this const has used.

Reply:
Thanks, will fix